src/components/Button.tsx

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components"
```

Comment:
Why not use CTA and extend it if need be instead of creating a new button?

Reply:
component removed.